src/main/kotlin/org/rust/ide/refactoring/changeSignature/RsChangeSignatureConfig.kt

```diff
@@ -0,0 +1,143 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring.changeSignature
+
+import com.intellij.lang.Language
+import com.intellij.psi.PsiElement
+import com.intellij.refactoring.changeSignature.ChangeInfo
+import com.intellij.refactoring.changeSignature.ParameterInfo
+import com.intellij.refactoring.changeSignature.ParameterInfo.NEW_PARAMETER
+import org.rust.ide.presentation.renderInsertionSafe
+import org.rust.ide.refactoring.extractFunction.dependTypes
+import org.rust.ide.refactoring.extractFunction.types
+import org.rust.lang.RsLanguage
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+import org.rust.lang.core.types.ty.Ty
+import org.rust.lang.core.types.ty.TyUnit
+import org.rust.lang.core.types.ty.TyUnknown
+import org.rust.lang.core.types.type
+
+/**
+ * This class just holds `config`, otherwise it is unimplemented.
+ * It is required by ChangeSignatureProcessorBase.
```

Comment:
```suggestion
 * This class just holds [config], otherwise it is unimplemented.
 * It is required by [com.intellij.refactoring.changeSignature.ChangeSignatureProcessorBase].
```